tailwind.config.js

```diff
@@ -107,10 +118,13 @@ module.exports = {
       },
       strokeWidth: {
         1.5: '0.09375rem',
+        2.5: '2.5',
```

Comment:
I guess, it should be `'2.5rem'`

Reply:
Actually not, the stroke itself works without `rem` or `px`:
![image](https://user-images.githubusercontent.com/49658988/192783971-66820644-5b6b-4e09-9acb-62b13cc73d67.png)
Somebody just converted the stuff above into `rem`, but I don't know why